src/components/contextual/pages/vebal/LMVoting/LMVoting.vue

```diff
@@ -54,6 +54,17 @@ const votingPeriodEnd = computed<number[]>(() => {
   return formattedTime;
 });
 
+const votingPeriodLastHour = computed<number[]>(() => {
+  const periodEnd = getVotePeriodEndTime();
+  const interval: Interval = { start: now.value, end: periodEnd };
+  const timeUntilEnd: Duration = intervalToDuration(interval);
+  if ((timeUntilEnd.days || 0) < 1 && (timeUntilEnd.hours || 0) < 1) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
All you really need to return here is the conditional because it resolves to true or false.
```suggestion
  return (timeUntilEnd.days || 0) < 1 && (timeUntilEnd.hours || 0) < 1
```